examples/morpheusvm/actions/transfer_test.go

```diff
@@ -142,6 +142,16 @@ func BenchmarkSimpleTransfer(b *testing.B) {
 			To:    to,
 			Value: 1,
 		},
+		ExpectedOutputs: func() []codec.Typed {
```

Comment:
Interesting - this test previously panicked. We don't run benchmarks at any cadence... feels like we should have some visibility into this... but it's unrelated to the PR.